practice_06.08.2019_System.IO/Task1_Folders/Folder.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+
+namespace practice_06._08._2019_System.IO.Task1_Folders
+{
+    public class Folder
+    {
+        public void ShowFoldersAndSize()
+        {
+            var path = @"C:\Windows\System32";
+            DirectoryInfo directory = new DirectoryInfo(path);
```

Comment:
No need to specify variable type explicitly when we use constructor.
Please use var instead.

Reply:
fixed